### PR TITLE
Add the global configuration to the docs

### PR DESCRIPTION
**Pull request by yak0d3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,17 +75,30 @@ Route::get('/home', 'FooController@bar')->from('ca', 'de', 'fr')->deny();
 
 > ***Note:*** This package uses [*stevebauman*][4]'s [location package][5], please refer to it's [official documentation][6] for a detailed guide on how to configure it correctly.
 
-#### Manual configuration
+### Global Configuration
```
Comment (nerg4l) [inline README.md:79]: Would it be beneficial to move the `php artisan vendor:publish [...]` part here?

File: README.md [hunk 2]
```diff
@@ -75,17 +75,30 @@ Route::get('/home', 'FooController@bar')->from('ca', 'de', 'fr')->deny();
...
 - [array] **`countries`**: The list of countries covered by the *geo-constraint*.
-- [string] **`strategy`**: Determines whether to allow or deny access, the value can only be **allow** or **deny**.
+- [boolean] **`allowed`**: Determines whether to allow or deny access from the configured countries.
 - [array] **`callback`** (optional): The callback that will be invoked once the access is denied and its arguments.
 
 Therefore, if you are more into verbosity, you can define your `GeoRoutes` in the following way:
 
 ```php
-Route::get([ 'geo' => ['countries' => ['us', 'ca'], 'strategy' => 'allow', 'callback' => [$myCallback, $myArgs]] ], function() {
+Route::get([ 'geo' => ['countries' => ['us', 'ca'], 'allowed' => true, 'callback' => [$myCallback, $myArgs]] ], function() {
```
Comment (nerg4l) [inline README.md]: I think this was accidentally moved here from #37

File: README.md (hunk 1 above)
Comment (yak0d3) [reply README.md:79]: I think it should be in the installation section where developers can see it before reading the whole docs, what about making a reference here for the "Publish the configuration" point?

File: README.md (hunk 2 above)
Comment (yak0d3) [reply README.md]: Oh my bad, thank you for noticing this, i was checking out from branch to another and ended up mixing things up lol.

File: README.md (hunk 1 above)
Comment (yak0d3) [reply README.md:79]: I have pushed a new commit, please take a second look and tell me what do you think.

**Review by nerg4l — APPROVED** [review]
Nice job

**Comment by yak0d3** [comment]
> 
> 
> Nice job

Thank you :smiley:

**Pull request merged**